src/frontend/overlay/src/components/atoms/ContestLabels.jsx

```diff
@@ -115,27 +117,52 @@ const TaskResultLabelWrapper2 = styled.div`
   color: #fff;
 `;
 
-// TODO: fts start
-const ICPCTaskResultLabel2 = ({ problemResult: r, ...props }) => {
+
+const StarIconWrap = styled.div`
+    position: absolute;
+    width: 33px;
+    height: 33px;
+    mask: url(${star});
+    background-position: center;
+    background-color: ${({ color }) => color};
+`;
+
+const StarIcon = ({ color }) => {
+    return <StarIconWrap color={color}/>;
+};
+
+const AttemptsOrScoreLabelWrapper = styled.div`
+    position: relative;
```

Comment:
Why is there a rouge position: relative?

Reply:
Doesn't it  work without the position: relative.

Reply:
If I remove it, the star will above the text

Reply:
Change the order of the elements in the html tree

Reply:
I just changed ```css position: relative``` to ```css position: absolute``` and removed ```css position: absolute``` from the StarIcon div.

Reply:
Ok, then just leave it